### PR TITLE
Change Language Selection(#1017)

### PR DESCRIPTION
**Pull request by Muskan7Tripathi**
# Description

Application for language selection option in this app.

Fixes (#1017 )

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
SS IMAGE
ss
[Language selection.zip](https://github.com/Swati4star/Images-to-PDF/files/7107489/Language.selection.zip)

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Swati4star** [comment]
@Muskan7Tripathi - There are merge conflicts in this PR. resolve those also.

File: README.md [hunk 1]
```diff
@@ -109,6 +111,3 @@ If you have any questions, feel free to ask [me](mailto:swati4star@gmail.com) ab
 ### Project Maintainers
 This project is founded and actively maintained by [Swati Garg](https://github.com/Swati4star/). For any sort of queries feel free to mail at swati4star@gmail.com.
 
-<a href="https://www.buymeacoff.ee/WmJepqz6H" target="_blank"><img src="https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png" alt="Buy Me A Coffee" style="height: auto !important;width: auto !important;" ></a>
```
Comment (Swati4star) [inline README.md:112]: why have you removed this?

File: .travis.yml [hunk 2]
```diff
@@ -4,11 +4,14 @@ android:
   components:
     - platform-tools
     - tools
-    - build-tools-28.0.3
+    - build-tools-30.0.3
```
Comment (Swati4star) [inline .travis.yml:7]: are these changes needed for your change? why?

File: app/proguard-rules.pro [hunk 3]
```diff
@@ -15,3 +15,4 @@
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
 #   public *;
 #}
+-dontwarn com.squareup.picasso.**
```
Comment (Swati4star) [inline app/proguard-rules.pro:18]: why this change?

File: app/build.gradle [hunk 4]
```diff
@@ -122,7 +124,7 @@ dependencies {
     testImplementation 'org.mockito:mockito-core:2.23.4'
     testImplementation 'org.robolectric:robolectric:4.3.1'
 
-    androidTestImplementation('com.android.support.test.espresso:espresso-core:2.2.2', {
+    androidTestImplementation('androidx.test.espresso:espresso-core:3.1.0', {
```
Comment (Swati4star) [inline app/build.gradle:127]: Is there a need to update these?

File: app/src/androidTest/java/swati4star/createpdf/util/ImageUtilsTest.java [hunk 5]
```diff
@@ -0,0 +1,124 @@
+package swati4star.createpdf.util;
```
Comment (Swati4star) [inline app/src/androidTest/java/swati4star/createpdf/util/ImageUtilsTest.java:1]: why have you added this file?

**Review by Swati4star — CHANGES_REQUESTED** [review]
There are 231 file chnages in this PR. Can you do a self review and check if all these are necessary for your feature?

**Comment by berlin4apk** [comment]
Sorry but For me, this PR don't look right!

Can you please split it to different clean new PRs?

**Pull request closed without merging**